src/test/java/org/apache/sysds/test/functions/mlcontext/MLContextTest.java

```diff
@@ -1965,28 +1964,4 @@ public void testNNImport() {
 			.getScalarObject("R").getDoubleValue();
 		Assert.assertEquals(1000, ret, 1e-20);
 	}
-
```

Comment:
Why were these tests removed?

Reply:
These seem to have been removed by accident. We put them back in.